openedx/openedx-platform
# fix: removed password from api response

## Description 
course live API was returning the password in plain text, now as requirements change it is modified to return an empty string instead of a password.

## Ticket
https://2u-internal.atlassian.net/browse/INF-527

## Testing instructions
In the course authoring app go to live settings and observe that password is not visible and API is not returning plain text client secret.
